spec/analist/checker_spec.rb

```diff
@@ -20,6 +20,16 @@
       it { expect(errors).to be_empty }
     end
 
+    context 'when annotating an unknown function call' do
+      let(:expression) { 'JSON.parse(1 + "bla")' }
+
+      it do
```

Comment:
It's common use to describe what it should do in the following form: 
`it 'gives a type error when such an error occures within a argument' do`
or something like that: https://github.com/howaboutwe/rspec-style-guide#rspec